sdks/java/core/build.gradle

```diff
@@ -49,11 +49,11 @@ interface for processing virtually any size data. This
 artifact includes entire Apache Beam Java SDK."""
 
 processResources {
+  inputs.property('version', version)
```

Comment:
I didn't realize we could use properties as inputs. Thanks for showing me the right way to do this :)

Reply:
:)